### PR TITLE
Fix systemd watchdog feature

### LINKED ISSUE #1353: Traefik shuts down when SystemD watchdog run
### What version of Traefik are you using (`traefik version`)?
1.2.1

### What did you do?
Install service file from: https://github.com/containous/traefik/blob/master/contrib/systemd/traefik.service

### What did you expect to see?
Running service

### What did you see instead?
Crashing service every 2s, with SystemD starting it again. Adjusting watchdog timings correlate with time between crashes

### PR DESCRIPTION
**Pull request by guilhem**
Commit coreos/go-systemd@0c088e introduce cleaning environment.
First usage of sdnotify (for type=notify) was clearing NOTIFY_SOCKET environment variable.
sdnotify in watchdog was unable to ping back.

Fix #1353

### CODE REVIEW

File: cmd/traefik/traefik.go [hunk 1]
```diff
@@ -244,20 +244,23 @@ func run(traefikConfiguration *server.TraefikConfiguration) {
...
 	}
-	t, err := daemon.SdWatchdogEnabled(true)
+	t, err := daemon.SdWatchdogEnabled(false)
 	if err != nil {
 		log.Error("Problem with watchdog", err)
 	} else if t != 0 {
 		// Send a ping each half time given
 		t = t / 2
+		log.Info("Watchdog activated with timer each ", t)
 		go func(interval time.Duration) {
```
Comment (emilevauge) [inline cmd/traefik/traefik.go]: In order to avoid panic, could you use `safe.Go` to launch the go routine here?

File: cmd/traefik/traefik.go [hunk 2]
```diff
@@ -244,20 +244,23 @@ func run(traefikConfiguration *server.TraefikConfiguration) {
...
 			tick := time.Tick(interval)
 			for range tick {
-				daemon.SdNotify(true, "WATCHDOG=1")
+				if ok, _ := daemon.SdNotify(false, "WATCHDOG=1"); ok {
+					log.Debug("Tick watchdog")
```
Comment (emilevauge) [inline cmd/traefik/traefik.go]: Shouldn't we log on error instead ?

Comment (guilhem) [inline cmd/traefik/traefik.go]: fixed :)

File: cmd/traefik/traefik.go (hunk 1 above)
Comment (guilhem) [inline cmd/traefik/traefik.go]: fixed

**Pull request merged**